src/Main.java

```diff
@@ -1,5 +1,91 @@
 public class Main {
     public static void main(String[] args) {
         System.out.println("Hello world!");
+        int clientOS = 0;
+        if (clientOS == 0) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        } else if (clientOS == 1) {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        } else {
+            System.out.println("Неизвестная операционная система");
+        }
+        // задача 1 окончена
+
+        clientOS = 1;
+        int clientDeviceYear = 2012;
+        if (clientOS == 0) {
+            if (clientDeviceYear < 2015) {
+                System.out.println("Установите облегченную версию приложения для iOS по ссылке");
+            } else {
+                System.out.println("Установите приложение для iOS по ссылке");
+            }
+        } else if (clientOS == 1) {
+            if (clientDeviceYear < 2015) {
+                System.out.println("Установите облегченную версию приложения для Android по ссылке");
+            } else {
+                System.out.println("Установите приложение для Android по ссылке");
+            }
+        } else {
+            System.out.println("Неизвестная операционная система");
+        }
+        //задача 2 окончена
+
+        int year = 2021;
+        if (year >= 1584 && year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
```

Comment:
```suggestion
        if (year >= 1584 && (year % 4 == 0 && year % 100 != 0 || year % 400 == 0)) {
```
надо скобки поставить